src/components/action-toolbar/action-toolbar-multi-action-button/__definition__.js

```diff
@@ -0,0 +1,7 @@
+import ActionToolbarMultiActionButton from './';
```

Comment:
These are probably not required if we are not using them

Reply:
These are required for the demo site (`Table` demo).